---a/net/minecraft/entity/ai/attributes/IAttribute.java
+++b/net/minecraft/entity/ai/attributes/IAttribute.java
@@ -1,7 +1,7 @@
 package net.minecraft.entity.ai.attributes;
 
-public interface IAttribute
-{
+public interface IAttribute {
+
     String getAttributeUnlocalizedName();
 
     double clampValue(double p_111109_1_);
@@ -11,4 +11,4 @@
     boolean getShouldWatch();
 
     IAttribute func_180372_d();
-}
+}
